docs/building.md

```diff
@@ -42,6 +42,7 @@ For Linux a few additional packages beyond the usual build essentials are needed
 - fontconfig library (`libfontconfig-dev` on debian based distributions)
 - (optional) Qt will be used when `qmake` is found in `PATH`
 - FFMPEG library `clang` `libavcodec-dev` `libavformat-dev` `libavutil-dev` `libavfilter-dev` `libavdevice-dev` `libasound2-dev` `pkg-config`
+- openssl (`libssl-dev` on debian based distributions)
```

Comment:
Hmm no. Slint itself doesn't require OpenSSL on the target. I think maybe one of the examples (the energy monitor). But it's not a general build requirement of Slint.

Reply:
I see there are other example specific items there, so no harm then:)